tests/conftest.py

```diff
@@ -1,42 +1,3 @@
-import os
-import pytest
-import httpretty
-from dotenv import load_dotenv
-from fastapi.testclient import TestClient
-
-from src.app import app
-from src.apis.schemas.header_schema import HeaderRequest
-
-
-@pytest.fixture
-def client():
-    """
-    This client can call the developed application
-    """
-
-    with TestClient(app) as test_client:
-        yield test_client
-
-
-@pytest.fixture
-def server():
-    """
-    This server can receive requests and mock the response
-    """
-
-    load_dotenv('default.env')
-
-    baseurl = 'http://testserver'
-    require_headers = HeaderRequest(
-        USERID_HEADER_KEY=os.environ.get('USERID_HEADER_KEY'),
-        GROUPS_HEADER_KEY=os.environ.get('GROUPS_HEADER_KEY'),
-        CLIENTTYPE_HEADER_KEY=os.environ.get('CLIENTTYPE_HEADER_KEY'),
-        BACKOFFICE_HEADER_KEY=os.environ.get('BACKOFFICE_HEADER_KEY'),
-    ).__dict__.items()
-
-    httpretty.enable(verbose=False, allow_net_connect=False)
-
-    yield baseurl, require_headers
-
-    httpretty.disable()
-    httpretty.reset()
+# pylint: disable=W0611
+from tests.fixtures.test_client import test_client
+from tests.fixtures.mock_server import mock_server
```

Comment:
This file is almost empty, do we need it?

Reply:
Yes, for the pytest framework you need to define global fixtures in one specific file. So I still kept them separate in dedicated files and imported them into the "master" file